pkg/command/service/clean.go

```diff
@@ -127,6 +139,14 @@ kperf service clean --namespace-prefix testns / --namespace nsname
 			} else {
 				fmt.Println("No service found for cleaning")
 			}
+
+			if cmd.Flags().Changed("verbose") {
+				fmt.Printf("[Verbose] Knative services cleaning finished.\n")
+				fmt.Printf("[Verbose] Cleaned namespaces number: %d\n", len(matchedNsNameList))
+				fmt.Printf("[Verbose] Cleaned Knative services number, success: %d, fail: %d\n",
```

Comment:
I think we can just keep it short `Cleaned Knative services` and remove number, it already looks straightforward with success/fail number.